lib/features/announcement/ui/discussion_forum/forum_page.dart

```diff
@@ -0,0 +1,170 @@
+                              fontSize: 16,
+                            ),
+                          ),
+                          const Text(
+                            '1 hour ago',
+                            style: TextStyle(
+                              color: Colors.grey,
+                              fontSize: 12,
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                  const SizedBox(height: 16),
+                  Text(
+                    'Discussion Thread ${index + 1}',
+                    style: const TextStyle(
+                      fontWeight: FontWeight.bold,
+                      fontSize: 18,
+                    ),
+                  ),
+                  const SizedBox(height: 16),
+                  const Text(
+                    'testing',
+                    style: TextStyle(
+                      fontSize: 16,
+                      color: Colors.grey,
+                      height: 1.5,
+                    ),
+                    maxLines: 2,
+                    overflow: TextOverflow.ellipsis,
+                  ),
+                  const SizedBox(height: 16),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Row(
+                        children: [
+                          IconButton(
+                            onPressed: () {},
+                            icon: const Icon(
+                              Icons.arrow_upward,
+                              color: Colors.grey,
+                              size: 16,
+                            ),
+                          ),
+                          const SizedBox(width: 4),
+                          const Text(
+                            '0',
+                            style: TextStyle(
+                              fontWeight: FontWeight.bold,
+                              color: Colors.grey,
+                            ),
+                          ),
+                          const SizedBox(width: 16),
+                          IconButton(
+                            onPressed: () {},
+                            icon: const Icon(
+                              Icons.comment,
+                              color: Colors.grey,
+                              size: 16,
+                            ),
+                          ),
+                          const SizedBox(width: 4),
+                          const Text(
+                            '2',
+                            style: TextStyle(
+                              fontWeight: FontWeight.bold,
+                              color: Colors.grey,
+                            ),
+                          ),
+                        ],
+                      ),
+                      const Text(
+                        'Last Updated: 1 hour ago',
```

Comment:
is this just a placeholder?  no need to internationalize it, if so

Reply:
yes, this one is for testing because the forum backend is not developed yet